### PR TITLE
Screenshots, Architecture, Cleanup, oh my!

### LINKED ISSUE #127: Add Architecture.md
We should document the architecture and how it works.

We can probably lift much of this from our internal planning document.

### LINKED ISSUE #160: How to specify frontend_api?
In the demo setup instructions frontend_api should be specified in editor launch parameters as `frontend_api=[IP_ADDRESS]` and in launcher app.ini parameters as `http://[IP_ADDRESS].sslip.io`. 
Confirming if this is intentional or they should be specified consistently? Thanks!

### PR DESCRIPTION
**Pull request by markmandel**
* Updates to the main README.md, including a screenshot, an overview, and an architecture diagram and description.
* Deleted the `/docs` folder - it didn't seem like it was being used any more.
* Added a description of the Director to the open match README.md

Closes #160, Closes #127

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,53 @@
...
+## Overview
+
 This multiplayer demo is a cloud first implementation of a global scale, realtime multiplayer game utilising
 dedicated game servers, utlising both Google Cloud's products and open source gaming solutions.
 
-## OAuth Authentication
+Projects and products utilised include:
 
-We need to manually set up the OAuth authentication, as unfortunately this cannot be automated.
+*   [Unreal Engine 5](https://www.unrealengine.com/) for the game client and server code.
```
Comment (markmandel) [inline README.md:12]: @kennycoder I lifted this from the draft I wrote in the blog post - WDYT?

File: services/open-match/README.md [hunk 2]
```diff
@@ -16,15 +16,19 @@ To align player latencies, the Match Function uses a separate [`MatchProfile`](h
...
+* Create 3 ticket matches from the sorted tickets, thereby grouping scores
 * Assign a match score that is simply the sum of the scores of each ticket, for use by the [Default Evaluator](https://open-match.dev/site/docs/tutorials/defaultevaluator/).
 
 The [Evaluator](https://open-match.dev/site/docs/guides/evaluator/) (part of Open Match Core) then chooses
 a match from the overlapping matches returned by the different profiles.
 
 ## Director
 
-TBD, write more when the Director is wired to Agones.
+The Director allocates a GameServer from an GKE Standard/Autopilot and Agones cluster hosted in the target region for a
```
Comment (markmandel) [inline services/open-match/README.md:27]: @zmerlynn does this work?

**Review by zmerlynn — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,24 +1,53 @@
...
+## Architecture
 
-Open the [Google OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent) for your project,
-and create an "External" App, and allowlist any users you wish to be able to login to your deployment of this game.
+The **Droid Shooter** game, is composed of a game client and dedicated server, and multiple backend services hosted
+around the globe.
 
-Open the [Google Credentials](https://console.cloud.google.com/apis/credentials) screen for your project, and click
-"+ CREATE CREDENTIALS", and create an "OAuth Client ID" of type "Web Application".
+![Architecture diagram](images/architecture.png)
```
Comment (markmandel) [inline README.md:27]: Would love feedback on this Architecture overview.

File: README.md (hunk 1 above)
Comment (kennycoder) [reply README.md:12]: SGTM!

**Review by kennycoder — APPROVED** [review]

**Pull request merged**